benches/benchmarks.rs

```diff
@@ -0,0 +1,51 @@
+use criterion::{black_box, Criterion, criterion_group, criterion_main};
```

Comment:
The benchmarks should be built during CI. Can you please add this to the PR?

Reply:
Added `--all-targets` to both the cargo check and clippy, I assume that's what you meant?

Reply:
Yes, that is fine.